### ISSUE TITLE
[BUG]: Xbox wireless controller (connected via USB) Not being detected by games but IS detected and mappable within PCSX2.

### ISSUE THREAD

**Issue by Spec-Ops-Gaming** [PCSX2/pcsx2#11387]
### Describe the Bug

I set-up the program following the guides, and have connected my controller (XBox wireless controller [connected via USB] ), I made sure to only have xinput checked, as well as attempting to use every possible variation of input options turned on and off as a form of troubleshooting, and I can map the controller perfectly fine, and it's functioning properly within the program as I can use it in the ESC menu from within games, But the games themselves do not seem to be detecting the controller at all.

I have also made sure to attempt an update on my controller drivers, and that didn't do anything as my drivers and the controllers themselves are up-to-date.

### Reproduction Steps

Plug in controller, then map the controller, then launch a game (specifically The Sims 2: Castaway), attempt to use the controller and find out that the controller does not register to the game, press ESC and find out that you can use the controller perfectly inside the ESC menu itself, repeat ad-nauseum

### Expected Behavior

I expected to plug my controller in, start PCSX2, map the controller and be able to use the controller in the game, with only MINOR issues.

I do really respect what you guys are doing, and I appreciate it so much as it brings such a wave of nostalgia when I imagine playing games from my childhood, But these controller issues are a pain in the neck.

### PCSX2 Revision

1.7.5886

### Operating System

Windows 10 (64bit)

### CPU

AMD Ryzen 7 5800X3D

### GPU

AMD Radeon RX 6700

### Logs & Dumps

there are no logs in the data folder for PCSX2

**Comment by Spec-Ops-Gaming** [comment]
it seems to be specifically with the sims 2 castaway, I have managed to get the controller to work on a different game now, but the sims 2 castaway is still refusing to recognize the controller no matter what I do

**Comment by srd4** [comment]
Hi I'm not part of the community in particular but just came to the repo to check what they were doing here and stumbled uppon your issue. As a last resort why don't you try to use [ds4](https://ds4-windows.com/get-started/)?

I know it sounds like a pain in the ass to use more software for something so simple like plugging in a controller but I bought a ps4 compatible chinese controller I was having trouble to play gta v with, and ds4 basically helps make windows believe that whatever cotroller you have is actually another, like an xbox one like yours or a ps4 etc. It's actually quite nice to use and allows for a lot of customization but also quite simple to configure if you prefer.

Was just using it to programm macros to buttons on the controller for my call center interpreter work at home job just yesterday, I hope it helps!

**Comment by stenzek** [comment]
This still sounds like a support issue (not bound correctly), so please use the Discord server/forums. You shouldn't need to use XInput, the default (SDL) works fine with xbox pads.

You can use padtest (drag/drop it over PCSX2) to verify your bindings: 
[padtest_ps2.zip](https://github.com/user-attachments/files/15814943/padtest_ps2.zip)

**Comment by Spec-Ops-Gaming** [comment]
> This still sounds like a support issue (not bound correctly), so please use the Discord server/forums. You shouldn't need to use XInput, the default (SDL) works fine with xbox pads.
> 
> You can use padtest (drag/drop it over PCSX2) to verify your bindings: [padtest_ps2.zip](https://github.com/user-attachments/files/15814943/padtest_ps2.zip)

Yeah no, it's not a configuration issue as everything is mapped correctly in the other games I have tried, and yes I did use your "padtest", and wouldn't you know it, everything works as intended....except, the sims 2 castaway, the only controls that game seems to recognize is keyboard and mouse, whether i use the "default (SDL)" that supposedly "works fine with xbox pads".  or I use any of the other possible input options, as i have already stated in my original Issue, that you decided to close without even attempting to ask me anything or seemingly read any of my original Issue posting. 

this isn't just "some dumbass who doesn't know how to use a computer", I genuinely laid out in my original issue posting that the SAME controller works perfect with other games, but does NOT work with the sims 2 castaway. as in, the game doesn't recognize the controller AT ALL, and I have tried both wired and wireless, Xbox and sony controllers...none of them actually work on this game.
